journalist_gui/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+name = "pypi"
+verify_ssl = true
+url = "https://pypi.python.org/simple"
+
+[requires]
+python_version = "3.5"
+
+[dev-packages]
+
+[packages]
+"pyqt5" = "*"
```

Comment:
Is this only used for the dev environment? I see there is also `requirements.txt`, which specifies the same packages. If this is ever used or might be used by admin workstations, I suggest pinning versions/hashes (by including pipfile.lock?)

Reply:
Hmm, good call. Yep this is also only used by the dev environment, but since we added the `Pipfile`, we _don't_ need `requirements.txt`... fixing

Reply:
Removed in a2653b12c75d7e6600d16f0e489da3ffab0e885e

Reply:
It is only used in the `dev environment`.